src/sql/workbench/contrib/notebook/browser/notebookViews/modal.component.html

```diff
@@ -0,0 +1,9 @@
+<!--
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+-->
+<div class="overlay" style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
```

Comment:
This doesn't seem to be used - you have the template in the modal.component.ts already